### PR TITLE
Feature/client side localization

### PR DESCRIPTION
**Pull request by alex-zyl**

### CODE REVIEW

File: Softeq.NetKit.Services.PushNotifications/Factories/PushRegistrationsFactory.cs [hunk 1]
```diff
@@ -10,8 +10,8 @@ namespace Softeq.NetKit.Services.PushNotifications.Factories
 {
     internal static class PushRegistrationsFactory
     {
-        private const string iOSTemplate = "{\"aps\":{\"alert\":{\"title-loc-key\":\"$(title_loc_key)\", \"title-loc-args\":[], \"loc-key\":\"$(body_loc_key)\", \"loc-args\":[], \"title\":\"$(title)\", \"body\":\"$(body)\"}, \"sound\":\"$(sound)\", \"badge\":\"#(badge)\", \"type\":\"#(type)\"}, \"data\":\"$(data)\"}";
-        private const string androidTemplate = "{\"notification\":{\"title_loc_key\":\"$(title_loc_key)\", \"title_loc_args\":[], \"body_loc_key\":\"$(body_loc_key)\", \"body_loc_args\":[], \"body\":\"$(body)\", \"title\":\"$(title)\", \"sound\":\"$(sound)\"}, \"data\":{\"type\":\"#(type)\", \"payload\":\"$(data)\"}}";
+        private const string iOSTemplate = "{\"aps\":{\"alert\":{\"title-loc-key\":\"$(title_loc_key)\", \"title-loc-args\":[\"$(title_loc_arg1)\", \"$(title_loc_arg2)\", \"$(title_loc_arg3)\"], \"loc-key\":\"$(body_loc_key)\", \"loc-args\":[\"$(body_loc_arg1)\",\"$(body_loc_arg2)\",\"$(body_loc_arg3)\"], \"title\":\"$(title)\", \"body\":\"$(body)\"}, \"sound\":\"$(sound)\", \"badge\":\"#(badge)\", \"type\":\"#(type)\"}, \"data\":\"$(data)\"}";
```
Comment (eugenekirvel) [inline Softeq.NetKit.Services.PushNotifications/Factories/PushRegistrationsFactory.cs:13]: can you please add unit test also

Comment (alex-zyl) [reply Softeq.NetKit.Services.PushNotifications/Factories/PushRegistrationsFactory.cs:13]: Done

File: Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.0.2-alpha2</Version>
+    <Version>1.1.0</Version>
     <Description></Description>
     <Authors>Softeq Development Corp.</Authors>
     <Company>Softeq Development Corp.</Company>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <PackageId>Softeq.PushNotificationService</PackageId>
-    <Product>Softeq.PushNotificationService</Product>
+    <PackageId>Softeq.NetKit.PushNotifications</PackageId>
```
Comment (eugepae) [inline Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj]: I think we shouldn't change packageId and product, just increment version

Comment (alex-zyl) [reply Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj]: reverted

**Review by eugenekirvel — APPROVED** [review]

File: Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.0.2-alpha2</Version>
+    <Version>1.1.0</Version>
     <Description></Description>
     <Authors>Softeq Development Corp.</Authors>
     <Company>Softeq Development Corp.</Company>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <PackageId>Softeq.PushNotificationService</PackageId>
+    <PackageId>SofteqSofteq.PushNotificationService</PackageId>
```
Comment (eugenekirvel) [inline Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj]: typo here

Comment (alex-zyl) [reply Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj]: damn copypaste
fixed

**Review by eugenekirvel — APPROVED** [review]

**Pull request merged**